src/data/builder-tools.js

```diff
@@ -270,14 +294,6 @@ export const Tags = {
     icon: null,
     color: "#4a69BD"
   },
-
-  // Koios
-  koios: {
-    label: "Koios",
-    description: "Koios and its integrations",
-    icon: null,
-    color: "#4d21b8"
-  },
```

Comment:
And what would you think of moving the `koios` tag definition back where it was in the source file, so it doesn't mistakenly look like this part of the code was changed?

Reply:
Yeah, I was wondering whether the new categories will cause any disagreements but that's how I feel is the right way to represent both Ogmios and Koios Java tools as best as we can.
Ogmios category will get bigger as we have already 2 more language-specific clients to interact with ogmios. so basically it's a big category of 5 tools including Ogmios Server. (https://ogmios.dev/typescript-client/)
I can add the other 2 as well but I'm not their maintainer.

Kotlin is a programming language. you can say it looks similar or resembles Java but in fact, it's a different programming language. Personally, I am familiar with Java but not with Kotlin and not with the syntax.

I noticed that this file was created in alphabetic order so I just wanted to give respect to the creator's first intention.
The alphabetic order wasn't kept as PR's kept merging. 
It won't be a problem to change it back if you insist.

Reply:
that's OK @edridudi I won't insist :)